stdlib/task/src/main/java/org/ballerinalang/stdlib/task/actions/Resume.java

```diff
@@ -60,6 +64,7 @@ public void execute(Context context) {
         try {
             task.resume();
         } catch (SchedulingException e) {
+            LOG.error(e.getMessage(), e);
```

Comment:
Wny do we need to log this as we are returning the error message to the user?

Reply:
The returning error is a custom error we created inside ballerina native functions. But we are logging that error with the error / exception from the Quartz, for debugging purposes.